sdk/core/switcher/entry.S

```diff
@@ -589,7 +590,7 @@ exception_entry_asm:
 	// See if we can find a handler:
```

Comment:
This comment doesn't seem to apply the succeeding lines. Is that updated in another PR?